Frontend/ui-customer/src/App.purs

```diff
@@ -366,6 +373,16 @@ data METRO_TICKET_SCREEN_OUTPUT = GO_TO_HOME_SCREEN_FROM_METRO_TICKET MetroTicke
 
 data PICKUP_INSTRUCTIONS_SCREEN_OP = GO_TO_HOME_SCREEN_FROM_PICKUP_INSTRUCTIONS
 
+data RIDE_SUMMARY_SCREEN_OUTPUT =     GO_TO_RIDE_REQUEST
+                                    | ACCEPT_SCHEDULED_RIDE String String
+                                    | UPDATE_ROUTE_INTERCITY Number Number Number Number
+                                    | UPDATE_ROUTE_RENTAL Number Number
+                                    | UPDATE_ROUTE_REGULAR Number Number Number Number
```

Comment:
can we combine all three in single screen output.

Reply:
i have deleted them as they are not being used customer side